content/docs/studio/troubleshooting.md

```diff
@@ -88,3 +96,50 @@ metrics or hyperparameters that you want to visualize.
 Refer to the [DVC documentation](https://dvc.org/doc) for help on making commits
 to a DVC repository. Instructions on how to specify custom files can be found
 [here](/doc/studio/user-guide/views/view-settings#configuring-view-settings).
+
+## View does not contain the columns that I want
+
+There are two possible reasons for this:
+
+1. **The required columns were not imported:** DVC Studio will import up to 200
+   columns (metrics, hyperparameters and files) from your Git repository. If
+   your repository has more than 200 columns, you should
+   [select the columns that are mandatory to import](/doc/studio/user-guide/views/view-settings#mandatory-columns).
+   DVC Studio will also import the unselected columns, but only up to a maximum
+   of 200 columns.
+
+   **What if there are more than 200 mandatory columns?**  
+   Currently DVC Studio cannot import over 200 columns. If you have a large
+   repository (with more than 200 mandatory columns), one solution is to split
+   the metrics/hyperparameters/files that you want to display over multiple
+   subdirectories in your git repository. For reach subdirectory, you can create
+   a new view in Studio that's limited to that subdirectory.
+
+   To create views in the case of a monorepo,
```

Comment:
This line also still needs to be updated now that we no longer mention monorepos in the paragraph above

Reply:
Done.